dbt/adapters/athena/config.py

```diff
@@ -0,0 +1,8 @@
+import pkg_resources
+from botocore import config
+
+
+def get_boto3_config() -> config.Config:
```

Comment:
This would probably benefit from a lrucache:

```
from functools import lru_cache

@lru_cache
def get_boto3_config() -> config.Config:
```

Reply:
yeah, you right, small nit that could speed up a bit code execution.

Reply:
Hello @jankatins , 

I just added @lru_cache but I did it like this `@lru_cache()` to continue supporting older versions of Python (i.e. 3.7 or below).

Thanks for the feedback!